instrumentation/runtime-telemetry-jfr/javaagent/build.gradle.kts

```diff
@@ -0,0 +1,18 @@
+plugins {
+  id("otel.javaagent-instrumentation")
+}
+
+otelJava {
+  minJavaVersionSupported.set(JavaVersion.VERSION_17)
+}
+
+dependencies {
+  implementation(project(":instrumentation:runtime-telemetry-jfr:library"))
+  compileOnly("io.opentelemetry:opentelemetry-sdk-extension-autoconfigure")
+}
+
+tasks {
+  test {
+    jvmArgs("-Dotel.instrumentation.runtime-telemetry-jfr.enabled=true")
```

Comment:
You can temporarily set system properties for the scope of a test via the [@SetSystemProperty](https://junit-pioneer.org/docs/system-properties/) annotation. Ignore if its more convenient to just enable jfr telemetry for all tests like this.

Reply:
That's pretty handy! I tried it out, but it doesn't seem to work in this situation. Maybe the system property is read before the test is actually started